.github/workflows/release_notifications_manual_tests.yml

```diff
@@ -83,10 +83,10 @@ jobs:
         uses: nearform-actions/github-action-notify-twitter@master
         with:
           message: ${{needs.test-release-notification-output.outputs.tweetText}}
-          twitter-app-key: ${{ secrets.TEST_TWITTER_API_KEY }}
-          twitter-app-secret: ${{ secrets.TEST_TWITTER_API_KEY_SECRET }}
-          twitter-access-token: ${{ secrets.TEST_TWITTER_ACCESS_TOKEN }}
-          twitter-access-token-secret: ${{ secrets.TEST_TWITTER_ACCESS_TOKEN_SECRET }}
+          twitter-app-key: ${{ secrets.TWITTER_API_KEY }}
```

Comment:
as an FYI, if you're going to make this change, then it has no difference from the other file and this can be deleted altogheter.

Reply:
well actually i forgot, the main difference between these two files is the use of the custom 'large example changelog' on line 43, 

so i guess its fine? it just means that this will post the test notification to the same channels as the normal release